resources/app.yaml

```diff
@@ -38,7 +38,6 @@ hooks:
             containers:
               - name: hook
                 image: monitoring-hook:0.0.1
-                args: ["update"]
```

Comment:
Use the command explicitly akin to `rollback` below even though it might be the default? This might help identify an inconsistency when the expectation about the default command does not hold.